recipes-mender/mender/files/mender.conf

```diff
@@ -1,7 +1,7 @@
 {
   "pollIntervalSeconds": 60,
-  "ServerURL": MENDER_SERVER,
-  "DeviceID": DEVICE_ID,
+  "ServerURL": "MENDER_SERVER",
+  "DeviceID": "DEVICE_ID",
```

Comment:
This was supposed to be hanged by the user before building image, but agree that having correct JSON makes sense here.